cli/src/commands/parse.rs

```diff
@@ -31,10 +44,12 @@ pub fn lockfile_types(add_auto: bool) -> Vec<&'static str> {
 pub fn handle_parse(matches: &clap::ArgMatches) -> CommandResult {
     let lockfiles = config::lockfiles(matches, phylum_project::get_current_project().as_ref())?;
 
-    let mut pkgs: Vec<PackageDescriptor> = Vec::new();
+    let mut pkgs: Vec<PackageDescriptorAndLockfilePath> = Vec::new();
 
     for lockfile in lockfiles {
-        pkgs.extend(parse_lockfile(lockfile.path, Some(&lockfile.lockfile_type))?.packages);
+        let parsed_lockfile = parse_lockfile(lockfile.path, Some(&lockfile.lockfile_type))?;
+        let parsed_packages = Vec::<PackageDescriptorAndLockfilePath>::from(parsed_lockfile);
+        pkgs.extend(parsed_packages);
```

Comment:
This seems unnecessary... Rust should be able to figure this out...

```suggestion
        pkgs.extend(parsed_lockfile.into());
```

If for some reason it can't, it's still cleaner to just specify the type of the variable and then call `.into()`

```suggestion
        let parse_packages: Vec<PackageDescriptorAndLockfilePath> = parse_lockfile.into();
        pkgs.extend(parse_packages);
```